src/widgetastic/widget/table.py

```diff
@@ -537,7 +537,8 @@ def _process_filters(self, *extra_filters, **filters):
             else:
                 column = filter_column
                 method = None
-                if isinstance(filter_value, re._pattern_type):
```

Comment:
As per my knowledge in `py3` we can go with `re.Pattern`.

Reply:
I think better to chat exception as its only for `py3.7` less than this we are good with `re._pattern_type`.
```
try:
    Pattern = re._pattern_type
except AttributeError:
    Pattern = re.Pattern
```

Reply:
I can do that, though personally I'd probably do the exception the other way round, so that the exception happens less as people use Python 3.7+ more:

```python
try:
    Pattern = re.Pattern
except AttributeError:
    Pattern = re._pattern_type
```